app/start_session.py

```diff
@@ -20,3 +36,22 @@ def start_session():
     )
     session.create_cube(table)
     return session
+
+
+def _get_content_storage_config() -> Mapping[str, Mapping[str, str]]:
```

Comment:
Can you move this function above `start_session` to not rely on hoisting?

```suggestion
def _get_user_content_storage_config() -> Mapping[str, Mapping[str, str]]:
```